package.json

```diff
@@ -2,6 +2,7 @@
   "name": "root",
   "private": true,
   "devDependencies": {
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
```

Comment:
remove this dependency, since ur not using it anywhere I think, 
**npm install** inside **root** folder,  and try again if it builds properly